lib/round.rb

```diff
@@ -0,0 +1,42 @@
+class Round
+  attr_reader :deck,
+              :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[turns.count]
+  end
+
+  def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    @turns << new_turn
+    new_turn
+  end
+
+  def number_correct
+    correct_turns.count
+
+  end
+
+  def number_correct_by_category(category)
+    correct_turns.select { |turn| turn.card.category == category }.count
```

Comment:
You're demonstrating a command of Ruby mechanics by method chaining the `count` method onto your `select` block. Not many people know that you can do that.